addons/l10n_ph/models/account_tax_template.py

```diff
@@ -0,0 +1,17 @@
+from odoo import fields, models
+
+
+class AccountTaxTemplate(models.Model):
+    _inherit = "account.tax.template"
+
+    atc = fields.Char("Philippines ATC")
```

Comment:
l10n_ph_atc will give more info on the field

Reply:
help info?

Reply:
I mean naming the field `l10n_ph_atc` will be better so that people know this field is part of this localization